backend/dal/PIMSContext.cs

```diff
@@ -1598,45 +1598,45 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
+                entity.Property(e => e.MailingAddrProvinceState).HasComment("Province or state, as required for mail to be sent to the owners mailing address.");
 
-                entity.Property(e => e.MailingAddrSite).HasComment("Site value, as required for mail to be sent to the owner?s mailing address.  e.g. Site 10.");
+                entity.Property(e => e.MailingAddrSite).HasComment("Site value, as required for mail to be sent to the owners mailing address.  e.g. Site 10.");
 
-                entity.Property(e => e.MailingAddrStreetDirectionPrefix).HasComment("Further refinement of the street name to facilitate location of building (pre-directional), as required for mail to be sent to the owner?s mailing address.");
+                entity.Property(e => e.MailingAddrStreetDirectionPrefix).HasComment("Further refinement of the street name to facilitate location of building (pre-directional), as required for mail to be sent to the owners mailing address.");
 
-                entity.Property(e => e.MailingAddrStreetDirectionSuffix).HasComment("Further refinement of the street name to facilitate location of building (post-directional), as required for mail to be sent to the owner?s mailing address.");
+                entity.Property(e => e.MailingAddrStreetDirectionSuffix).HasComment("Further refinement of the street name to facilitate location of building (post-directional), as required for mail to be sent to the owners mailing address.");
 
-                entity.Property(e => e.MailingAddrStreetName).HasComment("Road name assigned by a local government (municipality), as required for mail to be sent to the owner?s mailing address.");
+                entity.Property(e => e.MailingAddrStreetName).HasComment("Road name assigned by a local government (municipality), as required for mail to be sent to the owners mailing address.");
 
-                entity.Property(e => e.MailingAddrStreetNumber).HasComment("Street number assigned by a local government (municipality), as required for mail to be sent to the owner?s mailing address.");
+                entity.Property(e => e.MailingAddrStreetNumber).HasComment("Street number assigned by a local government (municipality), as required for mail to be sent to the owners mailing address.");
 
-                entity.Property(e => e.MailingAddrStreetType).HasComment("Street type assigned by a local government (municipality), as required for mail to be sent to the owner?s mailing address.");
+                entity.Property(e => e.MailingAddrStreetType).HasComment("Street type assigned by a local government (municipality), as required for mail to be sent to the owners mailing address.");
 
-                entity.Property(e => e.MailingAddrUnitNumber).HasComment("Apartment or Suite or Unit Number, as required for mail to be sent to the owner?s mailing address.");
```

Comment:
lol, ok.

Reply:
There is a problem with the encoding of these apostrophes. Might be changed later